src/config/firebaseRoar.js

```diff
@@ -9,7 +9,6 @@ const useSandbox = import.meta.env.VITE_FIREBASE_DATA_SOURCE === 'sandbox';
 const isStaging = import.meta.env.VITE_STAGING_BUILD === 'true';
 
```

Comment:
What do you think of adding the following and use `firebaseAppCheckDebugToken` in the `appConfig` declaration? 

```js
const firebaseAppCheckDebugToken = import.meta.env.VITE_APPCHECK_DEBUG_TOKEN || (import.meta.env.VITE_APPCHECK_DEBUG_TOKEN === 'true')
```

That way, we can set `VITE_APPCHECK_DEBUG_TOKEN=true` locally in order to generate our own debug token locally. Without this, the env var gets always interpreted as string and the Firebase SDK treats `true` as a debug token. 

With the expression above, we can control whether we want to generate a new token or set an existing one.

Reply:
I like this idea, thanks!